src/components/Synthetics/TradeBox/TradeBox.tsx

```diff
@@ -224,6 +231,77 @@ export function TradeBox(p: Props) {
   const [toTokenInputValue, setToTokenInputValue] = useSafeState("");
   const toTokenAmount = toToken ? parseValue(toTokenInputValue || "0", toToken.decimals)! : BigNumber.from(0);
 
+  const sortedIndexTokensWithPoolValue = useMemo(() => {
+    if (!marketsInfoData) return;
+    const markets = Object.values(marketsInfoData || {}).sort((a, b) => {
+      return a.indexToken.symbol.localeCompare(b.indexToken.symbol);
+    });
+    const marketsWithPoolValue = markets.reduce((acc, marketInfo) => {
+      if (marketInfo.isSpotOnly || marketInfo.isDisabled) {
+        return acc;
+      }
+
+      acc[marketInfo.indexTokenAddress] = marketInfo.poolValueMax;
+
+      return acc;
+    }, {});
+
+    return Object.keys(marketsWithPoolValue).sort((a, b) => {
+      return marketsWithPoolValue[b].gt(marketsWithPoolValue[a]) ? 1 : -1;
+    });
+  }, [marketsInfoData]);
+
+  const sortedLongAndShortTokens = useMemo(() => {
```

Comment:
hm i think this is better to do it in a cusom hook

probably in useAvailableTokenOptions and useAvailableMarketsOptions